docs: describe new access syntax (#451)
jina-ai/docarray: docs/fundamentals/dataclass/access.md

# Access Modality

After {ref}`constructing a multi-modal Document or DocumentArray <mm-construct>`, you can directly access your custom-defined
modalities by their names.

```{admonition} Return types
:class: seealso

Accessing a modality always returns a Document or a DocumentArray, instead of directly returning the data stored in them.

> If you are interested in the principles behind this design decision, you can learn more [here](TODO LINK TO BLOG POST).

Author: are we waiting for the blog post or should I remove this?

Reviewer: can u add the link ? or is not yet known?

# Access Modality

After {ref}`constructing a multi-modal Document or DocumentArray <mm-construct>`, you can directly access your custom-defined
modalities by their names.

```{admonition} Return types
:class: seealso

> Accessing a modality always returns a Document or a DocumentArray, instead of directly returning the data stored in them.
> 
> If you are interested in the principles behind this design decision, you can learn more [here](TODO LINK TO BLOG POST).

Author: ```suggestion
Accessing a modality always returns a Document or a DocumentArray, instead of directly returning the data stored in them.

This allows you, for example, to store embeddings directly in this Document.
```
